src/components/blocked-patron/BlockedPatron.test.ts

```diff
@@ -0,0 +1,77 @@
+import { TOKEN_LIBRARY_KEY } from "../../core/token";
+
+describe("Patron page", () => {
+  before(() => {
+    cy.window().then((win) => {
+      win.sessionStorage.setItem(TOKEN_LIBRARY_KEY, "random-token");
+    });
+  });
+
+  it("Patron not blocked", () => {
+    cy.intercept("GET", "**/external/agencyid/patrons/patronid/v2**", {
+      patron: {
+        blockStatus: null
+      }
+    });
+    cy.visit("/iframe.html?path=/story/apps-loan-list--loan-list-entry");
+    cy.get(".modal").should("not.exist");
+  });
+
+  it("Patron blocked E", () => {
```

Comment:
Is it possible to write what "blocked E" means? :)

Reply:
linked to enum